src/main.js

```diff
@@ -29,6 +29,7 @@ Vue.use(VueAnalytics, {
 Vue.use(VueAxios, axios);
 Vue.use(VueFire);
 Vue.use(VueI18n);
+Vue.use(require('vue-shortkey'));
```

Comment:
Can you use existing pattern just to keep things consistent?
```
import VueShortkey from 'vue-shortkey';

Vue.use(VueShortkey);

Reply:
Also, it's probably a good idea to disable shortcuts while typing on an input / textarea field

`Vue.use(VueShortkey, { prevent: ['input', 'textarea'] });`

Reply:
Resolved in next commit